src/arrayMethodSort.js

```diff
@@ -3,9 +3,42 @@
 /**
  * Implement method Sort
  */
+
+const compareAsStrings = (a, b) => {
+  const strA = String(a);
+  const strB = String(b);
+
+  if (strA > strB) {
+    return 1;
+  }
+  
+  if (strA === strB) {
+    return 0;
+  } 
+    
+  return -1;
+};
+
 function applyCustomSort() {
-  [].__proto__.sort2 = function(compareFunction) {
-    // write code here
+  [].__proto__.sort2 = function(compareFunction = compareAsStrings) {
```

Comment:
you can do the same by passing the default function
`[].__proto__.sort2 = function(
    compareFunction = (a, b) => String(a) > String(b)
  )`